docs: Added information for Sancho GovTool Beta (#113)
input-output-hk/sanchonet: docs/get-started/govtool.mdx

---
description: SanchoNet GovTool"
sidebar_position: 3
---

# SanchoNet GovTool

![SanchoNet GovTool Logo](../../static/img/sanchonet-govtool-header.png)

## What is the SanchoNet GovTool?

> The SanchoNet GovTool allows ada holders to experience now some of the governance features described in [CIP-1694](https://www.1694.io/) and to test governance features in SanchoNet in a guided and simple experience.

Reviewer: ```suggestion
The SanchoNet GovTool enables ada holders to experience some of the governance features described in [CIP-1694](https://www.1694.io/) and to test governance features in SanchoNet through a guided and straightforward experience. 
```

> The SanchoNet GovTool is open for beta testing now and it's accessible from [sanchogov.tools](https://sanchogov.tools/)

Reviewer: ```suggestion
The SanchoNet GovTool is currently open for beta testing and can be accessed at [sanchogov.tools](https://sanchogov.tools/).
```

## What is the Voltaire GovTool?

> The Voltaire GovTool, of which the SanchoNet GovTool is the testnet version, will enable ada holders to register to become a Delegated Representative (DRep), delegate voting power to DReps, and review and vote on governance actions.

Reviewer: ```suggestion
The Voltaire GovTool, with the SanchoNet GovTool serving as its testnet version, will empower ada holders to register as delegate representatives (DReps), delegate voting power to DReps, and review and vote on governance actions. 
```

Governance tooling is owned and maintained by the community.
With community approval, Intersect, through its committees and working groups, will play an important role in hosting and maintaining these governance tools.
Through active community involvement, the aim is to create the initial foundation layer together with different builders in the community.

> These governance tools will support the whole process, including both off-chain and on-chain, and as a whole they will support the implementation of the governance described in CIP-1694 and its adoption.

Reviewer: ```suggestion
These governance tools will support the entire process, encompassing both off-chain and on-chain aspects. As a whole, they will facilitate the implementation of the governance described in CIP-1694 and its adoption.
```

> Once this foundation layer is ready, the community will be able to further contribute to these governance tools by maintaining and improving existing features and pillars, creating new features or completely new areas either by themselves or supported by awards and grants facilitated by Intersect.

Reviewer: ```suggestion
Once this foundational layer is in place, the community will have the opportunity to enhance these governance tools. They will be able to achieve this by maintaining and improving existing features and pillars, creating new features, or establishing entirely new areas either independently or with support from awards and grants facilitated by Intersect.
```

With the freedom to test and refine Cardano’s next era of governance principles under one central apparatus, the GovTool helps the Cardano community forge a future governance framework that represents the community’s best democratic interests.

> ## How to participate in the Beta Testing?

Reviewer: ```suggestion
## How to participate in the beta testing?
```

> The Voltaire GovTool will be one of the first tools that allows ADA holders to experience the CIP-1694 Governance on Cardano.

Reviewer: ```suggestion
The Voltaire GovTool will be one of the first tools to allow ada holders to experience the CIP-1694 governance on Cardano.
```

The current version of this tool has been developed using initial feedback from the community and following what's described in CIP-1694.

> It's now important that the community tests this tool and provide valuable insights to further refine it and make it ready for future product use.

Reviewer: ```suggestion
It's now important that the community tests this tool and provides valuable insights to further refine it and make it ready for future product use.
```

Check the [beta testing to-do list](https://docs.sanchogov.tools/about/sanchonet-govtool-testing)
Check the [prerequisites to start](https://docs.sanchogov.tools/how-to-use-the-govtool/prerequsites)
Check the [quick/start guide](https://docs.sanchogov.tools/how-to-use-the-govtool/overview-quick-start)

## Related guides and resources

> [SanchoNet Govtool guides](https://docs.sanchogov.tools/)

Reviewer: ```suggestion
[SanchoNet GovTool guides](https://docs.sanchogov.tools/)
```

> Overall Governance Guides

Reviewer: ```suggestion
General governance guides
```

> Cardano Governance Page overview

Reviewer: ```suggestion
Cardano governance page overview
```

> Govtool overview

Reviewer: ```suggestion
GovTool overview
```
